### ISSUE TITLE
Enhancement: better deal with unrecognised faces

### ISSUE THREAD

**Issue by elpraga** [matiasdelellis/facerecognition#405]
Currently, if a face is marked as **Unknown** (using the **I don't know option**) it is being asked about every time.

This could be fine if there are two faces like that. Nevertheless, In my collection, there are some 30 people  do not really know / care to have groupped. Yet every time I want to add a new person I do care about , I have to go through all those.

If a possibility to **hide** a person was added, all those could be added as a **I don;t care** person, and subsequently hidden from the view.

I believe this is somewhat related to https://github.com/matiasdelellis/facerecognition/issues/404

**Comment by britzke** [comment]
As I understand, this feature is to IGNORE a group of RECOGNIZED faces which are unknown or by other reasons to be not named. This (possible more) groups of unnamed faces should not be presented in the list groups to be named (new persons). 

This feature would be nice to me, too.

**Comment by Cipioo** [comment]
Agreed, currently so I don’t have to cycle through those faces everytime I created a person named “unknown” and set them all to that, but I have no way of hiding these tags in my images.
